matlabCode/gsoc/gsoc_tenorio_2018

```diff
@@ -0,0 +1,25 @@
+%% Template Matlab script to create an BIDS compatible _events.tsv file
+% This example lists all required and some optional fields.
+%
+% MTenorio, 2018
+
+%%
+clear all
+root_dir = 'C:\Users\Marcus\Documents'; %Set manually your root directory
+project_label = 'templates';
+
+
+events_tsv_name = fullfile(root_dir,project_label,...
+    '_events.tsv'); %make sure you create a new blank file named '_events.tsv' on tamplates directory
```

Comment:
This file should go with appropriate naming into:
sub-01/ses-01/func/sub-01_ses-01_task-FullExample_run-01_events.tsv

Reply:
This file should have names consistent with other Matlab templates.